docs/developer-center/guides/01-Introduction.md

```diff
@@ -1 +1,39 @@
-## Introduction
\ No newline at end of file
+## Introduction
+
+Use these guides to learn about integrating CARTOframes into your **data science** workflows. Working inside of a Jupyter notebook, the guides walk through initial set-up and installation, reading data from CARTO into your Python session, writing data to your CARTO account, discovering data in our catalog, enriching data, using geocoding and isolines data services, and the basics of visualizing your data on a map.
+
+* [Quickstart]({{ site.url }}/developers/cartoframes/guides/Quickstart/)
+
+Learn the common steps data scientists follow in their **spatial analyses** and how CARTOframes library helps in each of them by following a **real use case**.
+
+* [Install CARTOframes in your Notebooks]({{ site.url }}/developers/cartoframes/guides/Install-CARTOframes-in-your-Notebooks/)
+
+Learn how to install the CARTOframes Python library in a **Jupyter Notebook**.
+
+* [Visualize your Data - Part 1]({{ site.url }}/developers/cartoframes/guides/Visualize-your-Data-Part-1/)
+
+Learn the basics to visualize data with CARTOframes: understand how to add different types of **layers**, change the **basemap**, configure the **viewport**.
+
+* [Visualize your Data - Part 2]({{ site.url }}/developers/cartoframes/guides/Visualize-your-Data-Part-2/)
+
+Explore deeper how to visualize data using built-in **helper methods**: use default styles, learn when to use helper methods, customize your visualization and create a layout of visualizations.
+
+* [Manage your data]({{ site.url }}/developers/cartoframes/guides/Manage-your-data/)
+
+Learn how to download, upload and update data with CARTOframes. Understand how to use **datasets**, **dataframes** and run **SQL** queries.
+
+* [Use Location Data Services]({{ site.url }}/developers/cartoframes/guides/Use-Location-Data-Services/)
+
+Learn to apply **geocoding** and **isolines** services to your data, and visualize the results.
+
+* [Discover Data]({{ site.url }}/developers/cartoframes/guides/Discover-Data/)
+
+Explore the and use **Data Observatory** methods to discover data in our **catalog**.
+
+* [Enrich Your Datasets]({{ site.url }}/developers/cartoframes/guides/Enrich-Your-Datasets/)
+
+Understand how to enrich **points** and **polygons** with data from our **catalog**, using dataset **variables** and applying **filters**. 
+
+* [Publish and share]({{ site.url }}/developers/cartoframes/guides/Publish-and-share/)
+
+Learn how to publish your results and share them with the world.
```

Comment:
Learn how to publish your results and configure their privacy to share them with the world.